consensus/enclave/api/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use mc_attest_enclave_api::{
 };
 use mc_common::ResponderId;
 use mc_crypto_keys::{CompressedRistrettoPublic, Ed25519Public, X25519Public};
-use mc_sgx_report_cache_api::ReportableEnclave;
+pub use mc_sgx_report_cache_api::ReportableEnclave;
```

Comment:
Might be worth organizing this in a separate grouping so it's easier to discover (since it's now `pub`).